fortinet_fortigate/plugin.spec.yaml

```diff
@@ -685,24 +685,26 @@ actions:
     title: Create Address Object
     description: Create an address object
     input:
-      ip:
-        title: IP
-        description: IP
+      host:
+        title: Host
+        description: The host to create. This can be an IP address, a CIDR or a domain name e.g. 198.51.100.100, 192.0.2.0/24, example.com
         type: string
         required: true
+        example: 198.51.100.100
         order: 1
-      cidr:
-        title: CIDR
-        description: CIDR
-        type: integer
-        required: true
-        default: 32
-        order: 2
       name:
         title: Name
-        description: Optional name to give this address object. If not provided, the name will be the IP address
+        description: Optional name to give this address object. If not provided, the name will be the IP address or domain name
         type: string
         required: false
+        example: MaliciousHost
+        order: 2
+      whitelist:
+        title: Whitelist
+        description: This list contains a set of network object that should not be blocked. This can include IPs, CIDR notation, or domains
```

Comment:
```suggestion
        description: This list contains a set of network object that should not be blocked. This can be an IP address, CIDR IP address e.g. 198.51.100.0/24, or a domain name
```